json/tbd_full_PSI.json

```diff
@@ -11,6 +11,7 @@
       "convex": 0.700,
       "balcony": 1.000,
       "party": 0.700,
+      "party": 0.800,
```

Comment:
This repeated key "party" should not be allowed right?

Reply:
Correct. [Here](https://github.com/rd2/tbd/blob/1f88452b05f6e5fc4654f4256e306a4fc4681a60/spec/tbd_tests_spec.rb#L1952), I left (myself) a reminder to address default JSON Schema validation with regards to repeated keys. Post-schema validation remains unavoidable.